routers/likes.py

```
@@ -0,0 +1,155 @@
+
+from schemas.like import LikeCreate, LikeResponse, LikeStatus
+from schemas.post import PostAllPostResponse
+from utils import auth, data
+
+router = APIRouter(prefix="/likes", tags=["likes"])
+
+
+@router.post("/", response_model=LikeStatus, status_code=status.HTTP_201_CREATED)
+async def create_like(
+        like_data: LikeCreate,
+        current_user: dict = Depends(auth.get_current_user)
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)